src/shared/components/DropdownInput.tsx

```diff
@@ -21,20 +23,25 @@ const DropdownInput = <T extends unknown>({
   selectValue,
   addOnClass = '',
   absoluteDropdown = true,
+  fillContainer = false,
 }: DropdownInputProps<T>): JSX.Element => {
-  const emptyOption: Option<T> = { label: placeholder, value: undefined };
   const [selectedOption, setSelectedOption] = useState<Option<T> | null>(
     initialValue || null,
   );
   const [dropdownOpen, setDropdownOpen] = useState(false);
-  const allOptions = [emptyOption, ...options];
+  const allOptions = placeholder
+    ? [{ label: placeholder, value: undefined }, ...options]
```

Comment:
don't include empty option if no placeholder provided (not sure if this is the best design and/or if we should remove the empty option altogether)